package.json

```diff
@@ -4,12 +4,25 @@
   "description": "Define a world and generate stories.",
   "main": "generateWorld.js",
   "scripts": {
-    "test": ""
+    "test": "node_modules/.bin/_mocha --",
+    "coverage": "node_modules/.bin/istanbul cover node_modules/.bin/_mocha --",
+    "view-coverage": "open ./coverage/lcov-report/index.html"
```

Comment:
`npm run test` runs the tests
`npm run coverage` generates coverage in a vc ignore dir
`npm run view-coverage` opens the entry coverage file in browser